src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -5435,29 +5435,29 @@ inline void compute_bilinear_interpolation_1c(T **srcRowPtrsForInterp, Rpp32s lo
 {
     Rpp32s loc1 = std::min(std::max(loc, 0), limit);
     Rpp32s loc2 = std::min(std::max(loc + 1, 0), limit);
-    *dstPtr = (U)std::nearbyintf((((*(srcRowPtrsForInterp[0] + loc1)) * bilinearCoeffs[0]) +     // TopRow 1st Pixel * coeff0
-                                  ((*(srcRowPtrsForInterp[0] + loc2)) * bilinearCoeffs[1]) +     // TopRow 2nd Pixel * coeff1
-                                  ((*(srcRowPtrsForInterp[1] + loc1)) * bilinearCoeffs[2]) +     // BottomRow 1st Pixel * coeff2
-                                  ((*(srcRowPtrsForInterp[1] + loc2)) * bilinearCoeffs[3])));    // BottomRow 2nd Pixel * coeff3
+    *dstPtr = (U)(((*(srcRowPtrsForInterp[0] + loc1)) * bilinearCoeffs[0]) +     // TopRow 1st Pixel * coeff0
```

Comment:
Why this changed is needed?

Reply:
when we use this nearby intf when it comes to run for f32 variants it is rounding off to 0 and 1 and it leads output mismatches
I checked the QA tests after removing the changes and the tests were running fine for all cases

Reply:
This issue of rounding off is not being observed for resize, but observe for pixelate?
Can you please check if it is leading to any 1 pixel mismatch for resize alone?

Reply:
@HazarathKumarM Pls verify this. We cannot have any mismatch in resize since its used in training heavily